tools/debian/control

```diff
@@ -0,0 +1,116 @@
+Source: cockpit
+Section: admin
+Priority: optional
+Maintainer: Michael Biebl <biebl@debian.org>
```

Comment:
Cockpit <cockpit@cockpit-project.org>, as @mvollmer used in his debian/changelog, would be suitable replacement